data-prepper-plugins/aws-lambda/README.md

```diff
@@ -1,4 +1,55 @@
 
+# Lambda Processor
+
+This plugin enables you to send data from your Data Prepper pipeline directly to AWS Lambda functions for further processing.
+
+## Usage
+```aidl
+lambda-pipeline:
+...
+  processor:
+    - aws_lambda:
+        aws:
+            region: "us-east-1"
+            sts_role_arn: "<arn>"
+        function_name: "uploadToS3Lambda"
+        max_retries: 3
+        invocation_type: "RequestResponse"
+        payload_model: "batch_event"
+        batch:
+            key_name: "osi_key"
+            threshold:
+                event_count: 10
+                event_collect_timeout: 15s
+                maximum_size: 3mb
+```
+
+`invocation_type` as RequestResponse will be used when the response from aws lambda comes back to dataprepper.
```

Comment:
nit:
`invocation_type` as RequestResponse is used when DataPrepper needs to process the response from AWS Lambda.

`invocation_type` as Event is used when the response from AWS Lambda is sent to an S3 bucket.